screens/RegisterScreen.js

```diff
@@ -1,11 +1,86 @@
-import React from "react";
-import { Text, View } from "react-native";
-import { generalStyles } from "../shared/styles";
+import React, { useRef, useState } from "react";
+import {
+  KeyboardAvoidingView,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import { generalStyles, formStyle } from "../shared/styles";
+import * as Texts from "../shared/text";
+
+export const RegisterScreen = props => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const lastNameInput = useRef(null);
+  const emailInput = useRef(null);
+  const passwordInput = useRef(null);
+
+  const registerHandler = () => {
+    props.navigation.navigate("Login");
+  };
 
-export const RegisterScreen = () => {
   return (
-    <View styles={generalStyles.center}>
-      <Text>REGISTER</Text>
-    </View>
+    <KeyboardAvoidingView behavior="padding" style={generalStyles.center}>
```

Comment:
Can you make the keyboard hide when I click on some empty space on the screen? Is that possible?

Reply:
Can you add validation for all text inputs? Or shall we do this as a separate task?

Reply:
idk, in future I'll read documentation and try to implement this functionality

Reply:
seperate task.